index.js

```diff
@@ -187,6 +188,15 @@ module.exports = {
 
   // After build output is ready, create a Percy build and upload missing build resources.
   outputReady: function(result) {
+    if (process.env.EMBER_CLI_TEST_COMMAND === 'true') {
+      this.createPercyBuild(result.directory);
+    }
+  },
+
+  // Create a Percy build and upload missing build resources.
+  createPercyBuild: function(buildOutputDirectory) {
+    this.createPercyBuildInvoked = true;
```

Comment:
Track that createPercyBuild has been invoked, so we can prevent it from being invoked a second time (especially for scenarios of missing tokens etc).

Reply:
I don't think this should be `this.createPercyBuildInvoked`, but rather `createPercyBuildInvoked`.